android/src/main/java/ly/count/android/sdk/react/CountlyReactNative.java

```diff
@@ -648,7 +648,26 @@ public void logException(ReadableArray args) {
         String exceptionString = args.getString(0);
         Exception exception = new Exception(exceptionString);
 
-        Countly.sharedInstance().crashes().recordHandledException(exception);
+        if (args.size() == 2 ) {
+            Countly.sharedInstance().crashes().recordHandledException(exception);
+            return;
+        }
+        
```

Comment:
I think we can remove the code from lin 651-655 and code still works as expected :)